### ISSUE TITLE
Include a non-VR version in the demo package

### ISSUE THREAD

**Issue by zachkinstner** [aestheticinteractive/Guitar-VR#16]
**PIZZABUG** at [YouTube](https://www.youtube.com/watch?v=iGT_iR5DGRo&lc=z13mvptzyvviglwqb04cedbxvy21vffamxw0k):

> Will this also be released for standard screens? It seems pretty cool, but I can't play on stage if I'm wearing a VR headset.

@zachkinstner response:

> The answer from Sean is correct -- it is difficult to play this without VR's one-to-one mapping between your actual and virtual hands. For other VR demos I've created, I sometimes include an "experimental" version for a table-mounted Leap Motion controller, so that's a possibility. 
> 
> Don't forget, the new AR headsets will be following right behind VR. That will be your sweet-spot for performing virtual instruments on stage.

**PUZZABUG** response:

> Even with a standard screen, you're still getting 1:1 mapping with movement, although not with screen space. I've noticed that I can easily accommodate for distortions in the motion controls and play as if my on screen hands are my real hands. I'm looking forward to try out both versions!

**Comment by zachkinstner** [comment]
I'm working on the project away from my office/Oculus, so I made progress with this tonight. I made a separate scene with the Leap in a table-mounted position/orientation, and moved the instrument slightly to fit better on the non-VR screen.

This setup works, but it's not very easy to use. Maybe with some more tweaking, or even requiring a [tilted Leap device](https://www.youtube.com/watch?v=BTUQW9QzNME), it could work better. In any case, it shouldn't be a problem to include an "experimental" Leap-only version with the app releases.
